syncserver/src/server/mod.rs

```diff
@@ -259,16 +262,16 @@ impl Server {
         let host = settings.host.clone();
         let port = settings.port;
         let deadman = Arc::new(RwLock::new(Deadman::from(&settings.syncstorage)));
-        let blocking_threadpool = Arc::new(BlockingThreadpool::default());
+        let blocking_threadpool = Arc::new(BlockingThreadpool::new(
+            settings.worker_max_blocking_threads,
+        ));
         let db_pool = DbPoolImpl::new(
             &settings.syncstorage,
             &Metrics::from(&metrics),
             blocking_threadpool.clone(),
         )?;
         let worker_thread_count =
             calculate_worker_max_blocking_threads(settings.worker_max_blocking_threads);
```

Comment:
In retrospect I don't know why we included `worker_` in this setting's name because it's the overall number of threads (not per worker). Whereas the naming of the `calculate_worker_max_blocking_threads` fn makes sense